tracker/event_tracker.go

```diff
@@ -15,15 +15,32 @@ type eventSubscription interface {
 	AddLog(log *ethgo.Log)
 }
 
-type eventTracker struct {
-	dataDir    string
-	config     *PolyBFTConfig
-	subscriber eventSubscription
-	logger     hcf.Logger
+type EventTracker struct {
+	dataDir      string
+	rpcEndpoint  string
+	contractAddr ethgo.Address
+	subscriber   eventSubscription
+	logger       hcf.Logger
 }
 
-func (e *eventTracker) start() error {
-	provider, err := jsonrpc.NewClient(e.config.Bridge.JSONRPCEndpoint)
+func NewEventTracker(
+	dataDir string,
```

Comment:
Is it intended to have different databases for different events being tracked? 

If that is the case, then we should parameterize name of db file:
`store, err := boltdbStore.New(filepath.Join(e.dataDir, "/deposit.db"))`

Reply:
Yes, so this escaped the refactor https://github.com/0xPolygon/polygon-edge/pull/1044/commits/90fa5454dd485d427f760d4478549faf087c772c